lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -836,10 +836,12 @@ getMicrosoftABI(CallingConvention CallConv, model::Architecture::Values Arch) {
     case CallingConvention::NearStdCall:
     case CallingConvention::NearSysCall:
     case CallingConvention::ThisCall:
-    case CallingConvention::ClrCall:
     case CallingConvention::NearPascal:
-    case CallingConvention::NearVector:
       return model::ABI::Microsoft_x86_64;
+    case CallingConvention::NearVector:
+      return model::ABI::Microsoft_x86_64_vectorcall;
+    case CallingConvention::ClrCall:
+      return model::ABI::Microsoft_x86_64_clrcall;
```

Comment:
You asked me to review this, but there's not much I have to comment here.
Was this related to some other open thread?

Reply:
It's a minor change. I asked you to review it because you never saw it before, just on the off-chance you see/know something I missed. If the code I asked to review looks good, feel free to just ignore the request (or mark it with `[x]` in the pad) - I'll assume you reviewed everything on the list unless you said anything otherwise.